metadata.json

```diff
@@ -8,9 +8,9 @@
     "summary": "Resource types for managing settings in HOCON files",
     "operatingsystem_support": [
       {
-        "operatingsystem": "CentOS",
+        "operatingsystem": "Debian",
```

Comment:
And I think we should get Travis and Jenkins Jobs set up on this repo before the 1.0 release, and at that point we can do the release under the PL namespace